templates/internet-of-things/smart-city.html

```diff
@@ -0,0 +1,705 @@
+      <p>
+        <a href="/core">About Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
+      <p>
+        <a href="/core#get-in-touch" class="p-button--positive">Contact us about Ubuntu Core</a>
+      </p>      
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/46f1b890-IoT_field_updates_from_Cloud.svg",
+        alt="",
+        width="500",
+        height="304",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>  
+</section>
+
+<section class="p-strip--suru is-dark">
+  <div class="u-fixed-width u-sv3">
+    <h2>Learn about some of our IoT solutions</h2>
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg",
+            alt="",
+            width="40",
+            height="35",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title p-heading--5">Data sheet</h4>
+        </div>
+      </div>
+      <p>
+        <a href="https://assets.ubuntu.com/v1/b2c770ea-Ubuntu+Core20+Datasheet.pdf" class="p-link--inverted p-link--external">Ubuntu Core 20</a>
+      </p>
+    </div>
+    <div class="col-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg",
+            alt="",
+            width="40",
+            height="35",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title p-heading--5">Data sheet</h4>
+        </div>
+      </div>
+      <p>
+        <a href="https://drive.google.com/file/d/1pZJfOiGQtiTcYdeGi2WeVM3zJuqlniMe/view?usp=sharing" class="p-link--inverted p-link--external">The IoT App Store</a>
+      </p>
+    </div>
+    <div class="col-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/3100aa6a-case-study-08.svg",
+            alt="",
+            width="40",
+            height="35",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title p-heading--5">Case study</h4>
+        </div>
+      </div>
+      <p>
+        <a href="/engage/drone-infrastructure-case-study-smartdrone" class="p-link--inverted">SmartDrone fast-tracks drone revolution with Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
+    </div>        
+  </div>
```

Comment:
It seems one row is missing here for the "Learn about some of our IoT solutions" section. There are 6 items total.